src/sys/socket/sockopt.rs

```diff
@@ -797,6 +797,17 @@ sockopt_impl!(
     libc::accept_filter_arg
 );
 #[cfg(target_os = "linux")]
+sockopt_impl!(
+    /// Defers completion on accept so only when data arrives.
```

Comment:
Could you try to reword this?  The first sentence isn't grammatically correct.  The second sentence's `x` refers to a variable that isn't anywhere in the prototype.  Probably "the number of seconds" would be better.  And how would one use this after listen(2), given that listen blocks?